etc/shaders/lovr.glsl

```diff
@@ -273,8 +265,26 @@ void initSurface(out Surface surface) {
   if (flag_clearcoatTexture) surface.clearcoat *= getPixel(ClearcoatTexture, UV).r;
 
   surface.clearcoatRoughness = Material.clearcoatRoughness;
+}
+
+void calcSurfaceData(inout Surface surface) {
+  surface.geometricNormal = surface.normal;
```

Comment:
Note: This sets `geometricNormal` to the bent normal, it's supposed to only use the raw normal from the mesh.